hibernate/hibernate-offline-startup/src/test/java/io/quarkus/ts/hibernate/startup/offline/test/DevModeHibernateOfflineStartupIT.java

```diff
@@ -0,0 +1,85 @@
+package io.quarkus.ts.hibernate.startup.offline.test;
+
+import static org.hamcrest.Matchers.is;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.DevModeQuarkusService;
+import io.quarkus.test.bootstrap.PostgresqlService;
+import io.quarkus.test.logging.Log;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.Dependency;
+import io.quarkus.test.services.DevModeQuarkusApplication;
+import io.quarkus.test.utils.AwaitilityUtils;
+import io.quarkus.ts.hibernate.startup.offline.pu.defaults.rest.DatabaseManagementResource;
+import io.restassured.http.ContentType;
+
+@QuarkusScenario
+public class DevModeHibernateOfflineStartupIT extends AbstractHibernateOfflineStartupIT {
+
+    private static final String APPLICATION_PROPERTIES = "src/main/resources/application.properties";
+
+    @Container(image = "${postgresql.latest.image}", port = 5432, expectedLog = "listening on IPv4 address", builder = FixedPortResourceBuilder.class)
+    static final PostgresqlService db = new PostgresqlService().setAutoStart(false);
+
+    @DevModeQuarkusApplication(dependencies = @Dependency(artifactId = "quarkus-jdbc-postgresql"))
+    static final DevModeQuarkusService app = (DevModeQuarkusService) new DevModeQuarkusService()
+            .withProperty("jdbc-url", "jdbc:postgresql://localhost:5432/mydb");
+
+    @Test
+    void testHotReload() {
+        // stop database, turn off the offline startup and expect a failure
+        if (!db.isRunning()) {
```

Comment:
The same check is in `startDbIfNotRunning()` method

Reply:
> The same check is in `startDbIfNotRunning()` method

good catch, I think it one point this did not inherit the abstract test method, but I didn't realize it; will fix it